CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 ## To be released
+- Welcome modal is now configurable through `welcomeConfig.js`
+- Welcome modal with feature switch header displays on startup
```

Comment:
Perhaps that was the wrong wording for it, but basically I meant to say that the header is something that is turned on/off through a flag in the `welcomeConfig.js` file